### DISCUSSION TITLE
basic doubt

### DISCUSSION THREAD

**Discussion by jadsongmatos** [rusqlite/rusqlite#1107] [Questions]
would there be any basic example of how to make a function that returns the query result in csv string format ?

**Comment by thomcc** [comment]
You're probably best using rusqlite in combination with `csv`: https://crates.io/crates/csv. An example for how to do this would be a bit involved since it would have to set up a table and all that.

I guess you might be able to do it with a function in sqlite too, but I wouldnt expect it to end up as robust as using a dedicated CSV serializer.
